datafusion/core/tests/sqllogictests/src/engines/datafusion/mod.rs

```diff
@@ -80,29 +77,8 @@ async fn run_query(ctx: &SessionContext, sql: impl Into<String>) -> Result<DFOut
     // Check if the sql is `insert`
     if let Ok(mut statements) = DFParser::parse_sql(&sql) {
         let statement0 = statements.pop_front().expect("at least one SQL statement");
-        if let Statement::Statement(statement) = statement0 {
-            let statement = *statement;
-            match statement {
-                SQLStatement::CreateTable {
-                    query,
-                    constraints,
-                    table_properties,
-                    with_options,
-                    name,
-                    columns,
-                    if_not_exists,
-                    or_replace,
-                    ..
-                } if query.is_none()
-                    && constraints.is_empty()
-                    && table_properties.is_empty()
-                    && with_options.is_empty() =>
-                {
-                    return create_table(ctx, name, columns, if_not_exists, or_replace)
-                        .await
-                }
-                _ => {}
-            };
+        if let Statement::Statement(_) = statement0 {
+            {}
```

Comment:
nit: it might be good in such cases leave your own comments in the PR saying why this change was done. I noticed people here doing such way and that looks cool :)